gen/dcos-config.yaml

```diff
@@ -96,9 +96,27 @@ package:
         "name": "dummy",
         "type": "bridge"
       }
-  - path: /etc/dcos/network/overlay
+  - path: /etc/dcos/network/overlay-master
     content: |
       {{ dcos_overlay_network }}
+  - path: /etc/dcos/network/overlay-agent-master
+    content: |
+      {
+        "master": "zk://leader.mesos:2181/mesos",
```

Comment:
Oh , whats the URI for the local zk ? I think need to point it to the local zk instead of the leader.